src/main/java/eu/bbmri_eric/negotiator/database/repository/DiscoveryServiceRepository.java

```diff
@@ -9,4 +9,6 @@
 public interface DiscoveryServiceRepository extends JpaRepository<DiscoveryService, Long> {
 
   Optional<DiscoveryService> findByUrl(String url);
+
+  Optional<DiscoveryService> findByName(String name);
```

Comment:
is this method necessary?

Reply:
fixed in fb760484e9b2dbec0c0c5b584decfbd60305ad4d